docs/conf.py

```diff
@@ -46,17 +47,20 @@
 
 # General information about the project.
 project = u'FaIR'
-copyright = u'2018, FaIR development team'
+copyright = u'2019, FaIR development team'
 author = u'FaIR development team'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'1.3'
+vlist = fair.__version__.split('.')
+vX = vlist[0]
+vY = vlist[1]
+version = vX + '.' + vY
```

Comment:
Despite the comments provided by sphinx, I think you actually want 

```python
from fair._version import get_versions  # requires that fair is installed on readthedocs

# The short X.Y version
version = get_versions()["version"]
# The full version, including alpha/beta/rc tags
release = version
```

The reason is that `get_versions()["version"]` automatically shortens on tags to vX.Y and for other commits you want the full thing (makes it easier to avoid confusion)